osbuild/remoteloop.py

```diff
@@ -54,8 +69,12 @@ def _message(self, msg, fds, sock):
         dir_fd = fds[msg["dir_fd"]]
         offset = msg.get("offset")
         sizelimit = msg.get("sizelimit")
+        lock = msg.get("lock")
```

Comment:
The defaults for the gets here should match the _create_device defaults, otherwise you'll end up passing None instead of False if it isn't included in msg.

Reply:
so you want me to set the defaults here too?

I set them in the caller below:

```
def device(              
        self,            
        filename,        
        offset=None,     
        sizelimit=None,  
        lock=False,      
        partscan=False,  
        read_only=False, 
        sector_size=512):

```

Reply:
ok I updated it to use defaults here too.